### PR TITLE
Update next section with latest changes

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Updated the next section to the latest changes

## Type of change

- Documentation Fix

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [ ] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx [hunk 1]
```diff
@@ -43,63 +28,160 @@ Features of the IOTA SDK:
...
+</TabItem>
+</Tabs>
+
+</TabItem>
+<TabItem value="typescript-node" label="Typescript (Node.js)">
+
+Check out our [Getting Started with Node.js](../getting-started/nodejs.mdx) guide for more detailed
+instructions.
+
+```bash npm2yarn
```
Comment (lucas-tortora) [inline next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:162]: this doesn't seem to be working locally

**Review by lucas-tortora — APPROVED** [review]
one small thing I noticed, but the file seems alright.

File: next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx (hunk 1 above)
Comment (Dr-Electron) [reply next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:162]: You mean the bpm2yarn?

Comment (Dr-Electron) [reply next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:162]: Should be fixed now

**Review by lucas-tortora — APPROVED** [review]

**Pull request merged**